firebase-installations/src/main/java/com/google/firebase/installations/remote/FirebaseInstallationServiceClient.java

```diff
@@ -205,12 +205,19 @@ private void writeFIDCreateRequestBodyToOutputStream(
 
   private static void writeRequestBodyToOutputStream(URLConnection urlConnection, byte[] jsonBytes)
       throws IOException {
-    try (OutputStream outputStream = urlConnection.getOutputStream()) {
-      if (outputStream == null) {
-        throw new IOException("Cannot send request to FIS servers. No OutputStream available.");
-      }
-      try (GZIPOutputStream gzipOutputStream = new GZIPOutputStream(outputStream)) {
-        gzipOutputStream.write(jsonBytes);
+    OutputStream outputStream = urlConnection.getOutputStream();
+    if (outputStream == null) {
+      throw new IOException("Cannot send request to FIS servers. No OutputStream available.");
+    }
+    GZIPOutputStream gzipOutputStream = new GZIPOutputStream(outputStream);
```

Comment:
Probably an unrelated question, feel free to just acknowledge:
Can this ever be NULL, similar to outputStream above?
Probably not, huh?

Reply:
It cant be null. I validated that. It throw NPE or returns a valid GzipOutputStream.